src/sources/dnstap/schema.rs

```diff
@@ -14,6 +23,267 @@ pub struct DnstapEventSchema {
 }
 
 impl DnstapEventSchema {
+    /// The message schema for the request and response message fields
+    fn request_message_schema_definition(&self) -> Collection<Field> {
+        let mut result = BTreeMap::new();
+        result.insert(
+            self.dns_query_message_schema().time().into(),
+            Kind::integer().or_undefined(),
```

Comment:
This is unrelated to your changes, but I think `opentelemetry` has timestamps in unix nanos, and we decided to convert those into `Kind::timestamp` when we parsed them - that's probably someone we should determine a consistent behavior for and apply it for all sources.